ot-node.js

```diff
@@ -167,6 +180,11 @@ class OTNode {
             this.logger.warn(`Watchdog service initialization failed. Error message: ${e.message}`);
         }
     }
+
+    stop() {
```

Comment:
Why do we have this method? Cannot find where are we using it

Reply:
In tests we are starting mode processes and we need a way to stop them once the test is finished. This method should be called in afterAll method in the bdd tests.